File: js/fibonacci.js
const fibonacci = (num) => {
    let fibArr = [0, 1]

    if (num == 0) {
        return fibArr[0];
    } else if (num == 1) {
        return fibArr[1];
    } else {
        for (let i = 0; i < num -1 ; i++) {
            fibArr.push(fibArr[i] + fibArr[i + 1])
        }
    }
    return fibArr[fibArr.length - 1]
}

module.exports = {fibonacci}

File: python/fibonacci.py
def fibonacci(n):
  arr = [0, 1]

  if n == 0:
    return arr[0]
  elif n == 1:
    return arr[1]
  else:
    for i in range(0, n - 1):
      arr.append(arr[i] + arr[i + 1])
    return arr[len(arr) - 1]